doc/modchannels.adoc

```diff
@@ -0,0 +1,90 @@
+= Modchannels
+
+Allows server side mods (SSMs) communication to client side mods (CSMs) and vice versa.
+
+== Server Side API
+
+=== Functions
+
+==== minetest.mod_channel_join(channel_name)
+
+* `channel_name`: string, modchannel name
+
+Returns a object for use with <<Methods>>. Creates the channel if it does not exist and joins the channel.
+
+==== minetest.register_on_modchannel_message(function(channel_name, sender, message))
+
+* `channel_name`: string, modchannel name (already joined)
+* `sender`: string, empty if from a ssm, player name if from a client
+* `message`: string, message
+
+=== Methods
+
+NOTE: `channel` here means the object returned by `minetest.mod_channel_join`.
+
+==== channel:leave()
+
+The server will leave the channel, meaning no more messages from this channel on `minetest.register_on_modchannel_message`
+
+TIP: set the channel to nil afterwords to free resources
+
+==== channel:is_writeable()
+
+* returns boolean, true if the channel is writeable
+
+==== channel:send_all(message)
+
+* `message`: string, limited to 65535 bytes
+
+Sends to all SSMs and CSMs on the channel.
+
+TIP: The message will drop if channel is not writable or invalid
+
+== Client Side API
+
+=== Functions
+
+==== minetest.mod_channel_join(channel_name)
+
+* `channel_name`: string, modchannel name
+
+Returns a object for use with <<Methods>>. Creates the channel if it does not exist and joins the channel.
+
+==== minetest.register_on_modchannel_message(function(channel_name, sender, message))
+
+* `channel_name`: string, modchannel name (already joined and recieved acknowledgement)
+* `sender`: string, empty if from a ssm, player name if from a client
+* `message`: string, message
```

Comment:
Similar to the one above, can be said to be equivalent.

Reply:
https://github.com/minetest/minetest_docs/pull/15/commits/3940703fc9579f4c929b80eb614a62b11b20dbbf